code/style.css

```diff
@@ -1,7 +1,574 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+/* MOBILE VIEW EXTRA SMALL DEVICES max-width: 360px */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: 'Raleway', sans-serif;
+}
+
+/* HEADER/HERO IMAGE NAVBAR CONTAINER WITH LOGO, HAMBURGER & NAV MENU*/
+/* 
+.container {
+  width: 100%;
+  align-self: flex-end;
+} */
+
+.navbar-container {
+  padding: 20px 20px;
+  margin: auto;
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: space-between; 
+}
+
+.navbar-container nav {
+  display: none;
+} 
+
+.title {
+  padding: 20px;
+  margin: 5px;
+  width: 100px;
+  background: rgb(230, 243, 229);
```

Comment:
I would suggest either choosing `rgb ` or hex codes for colors rather than using both in different places

Reply:
Ok will do :)